packages/react-generator/src/expressions/react-component-input.ts

```diff
@@ -85,6 +85,10 @@ export class ComponentInput extends BaseComponentInput {
     return members.some((member) => member.isNested);
   }
 
+  membersFromTypeDeclarationIgnoreMembers(): string[] {
+    return [...super.membersFromTypeDeclarationIgnoreMembers(), 'component', 'render', ''];
+  }
+
```

Comment:
will work only for case where template property is named 'template'
if it will be named for example 'someTemplate' this members will have names 'someComponent' and 'someRender'